data/sql/updates/pending_db_world/asd.sql

```diff
@@ -0,0 +1,4 @@
+-- maexxna portals
+REPLACE INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `VerifiedBuild`, `zoneId`, `areaId`) VALUES 
```

Comment:
ZoneId and areaId doesn't exist in our creature table. Fix this and read the wiki that i commented above. Thanks!!